server/base/src/main/java/org/apache/accumulo/server/AccumuloDataVersion.java

```diff
@@ -36,6 +36,11 @@
  */
 public class AccumuloDataVersion {
 
+  /**
+   * version (11) reflects removal of replication starting with 3.0
+   */
+  public static final int REMOVE_REPLICATION = 11;
```

Comment:
I suspect that this will be just one of several things that this upgrade will perform. It might be better to rename this to something more broadly applicable, such as `REMOVE_DEPRECATIONS_FOR_VERSION_3`. Can always enumerate the details in the javadoc, if desired.

Reply:
Changed in b3ca1d9b65